cmd/headscale/headscale.go

```diff
@@ -39,9 +39,10 @@ https://gitlab.com/juanfont/headscale`,
 func main() {
 	err := cli.LoadConfig("")
 	if err != nil {
-		log.Fatalf(err.Error())
```

Comment:
How will this affect normal operation, i.e. not the `init` path? Do we have another check further in the code for a non-existent config?

Reply:
I would also see this change as important, because you won't be able to even run the help command without a proper config setup up. 
I would agree to @cure some more check in the code would be needed to have a valid configuration for the other commands, beside help, version and init.